app/public/dist/client/locales/fr/translation.json

```diff
@@ -1163,12 +1163,12 @@
     "WATTREL": "Zapétrel",
     "KILOWATTREL": "Fulgulairo",
     "STANTLER": "Cerfrousse",
-    "BURMY_PLANT": "Cheniti (Cape Plante)",
-    "BURMY_SANDY": "Cheniti (Cape Sable)",
-    "BURMY_TRASH": "Cheniti (Cape Déchet)",
-    "WORMADAM_PLANT": "Cheniselle (Cape Plante)",
-    "WORMADAM_SANDY": "Cheniselle (Cape Sable)",
-    "WORMADAM_TRASH": "Cheniselle (Cape Déchet)",
+    "BURMY_PLANT": "Cheniti (Plante)",
```

Comment:
Cape Plante is the right name of this form, I'm sure of that because it's my language

Reply:
Checked through wikis as stated in commits; Lycanroc's form names stay changed because the sites say so

Reply:
honestly this is more of a "overthinking" moment as EN lacks the "Cloak" in brackets (corresponds to "Cape" in FR and "Manto" in ES), my bad